src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -20,8 +20,8 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the loan identified by the index number used in the displayed loan list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "Parameters: INDEX (must be a positive integer) and PASSWORD\n"
+            + "Example: " + COMMAND_WORD + " i/1 x/12345";
```

Comment:
The "parameters" and "example" are not consistent with the other commands. Either do
`"Parameters: INDEX (must be a positive integer) PASSWORD\n"
+ "Example: " + COMMAND_WORD + " 1 12345"`
or
`"Parameters: " + PREFIX_INDEX + "INDEX (must be a positive integer) " + PREFIX_PASSWORD + "PASSWORD\n"
+ "Example: " + COMMAND_WORD + " i/1 x/12345"`